legacy/composer.json

```diff
@@ -16,20 +16,20 @@
   "repositories": [
     {
       "type": "vcs",
-      "url": "https://github.com/jooola/propel1"
+      "url": "https://github.com/libretime/propel1"
```

Comment:
Does this require a lock file update?

Reply:
Yes it does. I have update the lock file as well.